core/src/apps/management/reset_device.py

```diff
@@ -108,7 +111,11 @@
 
     # generate the mnemonics
     mnemonics = slip39.generate_single_group_mnemonics_from_data(
-        secret, storage.device.get_slip39_identifier(), threshold, shares_count
+        encrypted_master_secret,
+        storage.device.get_slip39_identifier(),
+        threshold,
+        shares_count,
+        storage.device.get_slip39_iteration_exponent(),
```

Comment:
this function grew a parameter.
does the storage know the iteration exponent? did we save it before? was this another bug?

Reply:
1. Intentionally.
1. Yes, we store it above https://github.com/trezor/trezor-firmware/blob/cb9445461835e861e3e6254b3ad1089c3c61895c/core/src/apps/management/reset_device.py#L55 .
1. To some extent. We use the default iteration exponent everywhere, so this was fine. However, in future, this might cause problems if we would support resets with different iteration exponents (which we do not now).